src/diffusers/pipelines/__init__.py

```diff
@@ -46,6 +46,7 @@
         "AutoPipelineForInpainting",
         "AutoPipelineForText2Image",
     ]
+    _import_structure["modular_pipeline"] = ["ModularPipeline"]
```

Comment:
Need to add `components_manager` and at parent level because in the example we are using

```python
from diffusers import ComponentsManager
```